pkg/detectors/stripo/stripo.go

```diff
@@ -0,0 +1,88 @@
+package stripo
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// JWT token
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"stripo"}) + `\b(eyJhbGciOiJIUzI1NiJ9\.[0-9A-Za-z]{130}\.[0-9A-Z-a-z\-_]{43})\b`)
```

Comment:
nit: Just for future regexes I noticed that when you create this pattern `[0-9A-Z-a-z\-_]` it includes an extra `-` and a redundant escape. We can just get away with `[0-9A-Za-z-_]`. No need to adjust this one, I can update it as I merge in main and resolve the conflicts, just for future reference.

Reply:
I'd recommend either doing `[0-9A-Z-az\-_]` or `[0-9A-Za-z_-]` out of an abundance of caution. Putting just `-` between characters can inadvertently create a range. For instance, `[a-z0-9_-~]` matches any symbol with a character value between `_` (95) and `~` (126).